SPECS/moby-containerd/generate-sources.sh

```diff
@@ -5,8 +5,8 @@
 # Git clone is a standard practice of producing source files for moby-* packages.
 
 CONTAINERD_REPO=https://github.com/containerd/containerd.git
-CONTAINERD_COMMIT=814b7956fafc7a0980ea07e950f983d0837e5578
-VERSION=1.3.4+azure
+CONTAINERD_COMMIT=05f951a3781f4f2c1911b05e61c160e9c30eaa8e
```

Comment:
Ask: since we're already here, could we modify the `tar` command below to always generate a tarball with the same hash regadless of where and when you run it? I've recently hit an issue (see: `coredns-1.6.7.spec`) where I was trying to regenerate a source tarball locally and always ended up with a different hash, making our signatures worthless for such sources and making easier to miss a supply chain attack.

My suggestion for this `tar` command (the last line is a copy-paste of the current arguments):

```bash
tar --sort=name \
    --mtime="2021-04-26 00:00Z" \
    --owner=0 --group=0 --numeric-owner \
    --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
    -C /build/containerd-src -czf ./moby-containerd-${VERSION}.tar.gz .

#   NOTES:
#       - You require GNU tar version 1.28+.
#       - The additional options enable generation of a tarball with the same hash every time regardless of the environment.
#         See: https://reproducible-builds.org/docs/archives/
#       - The value of "--mtime" is an arbitrary date, which shouldn't be changed to make sure we can always regenerate older tarballs if necessary.
```

Reply:
Sure, I'll update this, as well as the other moby-* generate-sources.sh scripts.

Reply:
updated. I will hold off on the other scripts for a separate PR. (updating those existing source tarballs will actually be a breaking change so I want to be careful)